### PR TITLE
The server now lives in polymer-editor-service.

### PR DESCRIPTION
**Pull request by rictic**
This greatly simplifies this package, and sets the stage for sharing even more code between editor plugins, and *especially* with other plugin authors.

 - [x] CHANGELOG.md has not been updated, as this is an entirely internal change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "polymer-ide",
```
Comment (justinfagnani) [inline package.json]: is this really the npm package name?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "description": "Provides linting, autocompletion, and more for web components.",
+  "author": "Polymer Authors",
+  "license": "MIT",
+  "version": "0.1.2",
+  "icon": "polymer.png",
```
Comment (justinfagnani) [inline package.json]: We should put in an icon request.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "polymer-ide",
+  "description": "Provides linting, autocompletion, and more for web components.",
+  "author": "Polymer Authors",
+  "license": "MIT",
+  "version": "0.1.2",
```
Comment (justinfagnani) [inline package.json]: Was this ever actually published? If so, this is a at least a patch change because it now has a new dependency

Comment (rictic) [inline package.json]: Agreed, but we'll stamp the new version at release time, yeah?

File: package.json (hunk 2 above)
Comment (rictic) [inline package.json]: +1, I'll send an email

**Pull request merged**